src/test/java/guitests/IssuePanelTests.java

```diff
@@ -34,6 +34,7 @@
 public class IssuePanelTests extends UITest {
 
     private static final int EVENT_DELAY = 1000;
+    private UtilMethods util = new UtilMethods();
```

Comment:
Every test class which does keyboard input is already a subclass of UITest, so you don't need this object -- overriding `type` in UITest would allow you to call it as if it had been defined on all the test classes.